app/bundles/PageBundle/Model/TrackableModel.php

```diff
@@ -245,11 +245,8 @@ public function parseContentForTrackables($content, array $contentTokens = [], $
             'first_pass' => [
                 // Remove internal attributes
                 // Editor may convert to HTML4
-                'mautic:disable-tracking=""' => '',
-                // HTML5
-                'mautic:disable-tracking' => '',
+                'data-mautic-tracking="disable"' => '',
```

Comment:
Maybe we should keep the previous attributes for BC. Some users use it (https://github.com/mautic/mautic/issues/2927#issuecomment-293379553)